spec/helpers/canonical_rails/tag_helper_spec.rb

```diff
@@ -14,12 +14,12 @@
   # Default behavior
   describe 'w/ default config' do
     it 'should infer the domain name by looking at the request' do
-      expect(helper.canonical_host).to eq 'www.alternative-domain.com'
+      expect(helper.send(:canonical_host)).to eq 'www.alternative-domain.com'
```

Comment:
If you need to test code with `send` it means that there is something wrong with the code.

Reply:
Because I consider that it will be better to move some methods to private scope. Maybe now we can remove the tests which related to them. The tests for `canonical_href`, `canonical_tag` are much enough.

Reply:
I have push a new commit https://github.com/jumph4x/canonical-rails/pull/60/commits/53dd833ed991885bb11b3ba00270cba13901fc03